cloudinit/config/cc_users_groups.py

```diff
@@ -41,6 +41,7 @@
     - ``no_user_group``: Optional. Do not create group named after user.
       Default: false
     - ``passwd``: Hash of user password
+      Note: ``passwd`` has no effect on existing users.
```

Comment:
As discussed in the bug, this isn't limited only to the `passwd` field: none of the configuration is applied for existing users. Could you move your note to below the configuration, alongside the other notes, and reword it to refer to all the configuration?

Reply:
@OddBloke - I have a doubt here. You are saying none of these configs apply on existing users but
```
ssh_authorized_keys
sudo
lock_passwd
ssh_redirect_user
```

By looking at code, the above configs seem to be applicable  on existing users? Please clarify the confusion.

Reply:
Ah, yeah, you are correct, thanks for the pushback. I was getting mixed up between `add_user` and `create_user`. That's very confusing!

I think we should have a top-level message that notes that only those keys will be processed for existing users. What do you think?

Reply:
Agree, perhaps a note which says that following configs don't take effect on existing users or the other way round.